chihacknight/decarbonize-my-state
# Update Take Action with Feedback & Standardize Site Titles

## Overview

Overhauls the Take Action page to remove the Google Form, tweak the Climate Cabinet links to not jump straight to ActBlue, and makes the savings list more visually appealing like so:

![Screenshot from 2023-01-10 22-35-49](https://user-images.githubusercontent.com/3187531/211718754-ded77580-4bfe-4036-9942-b9cb8bdce479.png)

Note that this table now shows _tax credits_ (which are already in effect) rather than _rebates_, which are rolling out at... some point 🤷🏻‍♂️ 

Also a few minor fixes

- Fixes the title of the power section graph (previously said "MWs of targetGeneration Built", now "MWs of Wind & Solar Built")
- Standardizes site titles to "{Page Title} | Decarb My State" to make bookmarks and page titles clearer (e.g. "Decarb My State" for the homepage, "Take Action | Decarb My State", "Illinois | Decarb My State").

Closes #167 

### Demo

See description.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

- Test with Axe DevTools to confirm that there are no accessibility issues (besides Bootstrap success button's not being contrast compliant)
- Test on desktop and mobile to make sure `/take-action` is readable
